tests/sqlparser_common.rs

```diff
@@ -580,7 +580,7 @@ fn parse_select_count_wildcard() {
 
 #[test]
 fn parse_select_count_distinct() {
-    let sql = "SELECT COUNT(DISTINCT + x) FROM customer";
+    let sql = "SELECT COUNT(DISTINCT +x) FROM customer";
```

Comment:
Ah, I was confused about this for a while, but then I see that `+x` is actually a unary op, so it makes sense to display it as `+x` rather than `+ x` 👍